examples/authorize-beta/src/main/java/com/dropbox/core/examples/authorize_beta/Main.java

```diff
@@ -32,8 +31,8 @@ public static void main(String[] args) throws IOException {
         // Only display important log messages.
         Logger.getLogger("").setLevel(Level.WARNING);
 
-        if (args.length != 2) {
-            System.out.println("Usage: COMMAND <app-info-file> <auth-file-output>");
```

Comment:
nit: since fatal, System.err.println is more clear in this case

Reply:
Changed to err